src/graphql/handler.js

```diff
@@ -31,109 +34,214 @@ import { verifyEDLJwt } from '../utils/verifyEDLJwt'
 
 import { getCollectionsById } from '../dataloaders/getCollectionsById'
 
-// Creating the server
 const server = new ApolloServer({
   // Passing types and resolvers to the server
   typeDefs,
   resolvers,
+  plugins: [ApolloServerPluginLandingPageLocalDefault({ embed: false, footer: false })]
+})
 
-  // Initial context state, will be available in resolvers
-  context: async ({ event }) => {
-    const { body, headers } = event
-
-    const { operationName } = JSON.parse(body)
-
-    // If the query is the IntrospectionQuery, return out of this method
-    // The IntrospectionQuery is used when the playground has schema polling
-    // enabled. Returning out of this method for those calls saves API
-    // requests to URS and database calls
-    if (operationName === 'IntrospectionQuery') return null
-
-    const {
-      authorization: bearerToken,
-      'client-id': clientId,
-      'x-request-id': requestId
-    } = downcaseKeys(headers)
-
-    // Context object that we'll provide to each resolver
-    const context = {}
-
-    // Default headers to be sent with every external request
-    const requestHeaders = {
-      'CMR-Request-Id': requestId || uuidv4()
-    }
+export default startServerAndCreateLambdaHandler(
+  server,
+  handlers.createAPIGatewayProxyEventRequestHandler(),
+  {
+    context: async ({ event }) => {
+      const { body, headers } = event
+
+      const { operationName } = JSON.parse(body)
+
+      // If the query is the IntrospectionQuery, return out of this method
+      // The IntrospectionQuery is used when the playground has schema polling
+      // enabled. Returning out of this method for those calls saves API
+      // requests to URS and database calls
+      if (operationName === 'IntrospectionQuery') return null
+
+      const {
+        authorization: bearerToken,
```

Comment:
I think this could be a EDL token or Launchpad token. Seems odd to call it `bearerToken` if its possible that it is not a bearer token specifically.